src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/RabbitMq/BasicPublishWrapper.cs

```diff
@@ -22,7 +22,7 @@ public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
 
         public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
         {
-            int rabbitVersion = GetRabbitMQVersion(instrumentedMethodCall);
+            int rabbitVersion = RabbitMqHelper.GetRabbitMQVersion(instrumentedMethodCall);
```

Comment:
Can we cache the version here too? Or is there a use case where there might be more than one in use?

Reply:
I had caching setup, but I must have removed it at one point.  I will re-add it!